sirepo/package_data/static/js/raydata.js

```diff
@@ -809,15 +828,39 @@ SIREPO.app.directive('scansTable', function() {
             };
 
             $scope.columnIsSortable = (column) => {
-                return column !== 'stop';
+                if (['stop'].includes(column)) {
+                    return false;
+                } else if ($scope.scans.length > 0 && $scope.scans[0][column] !== null && typeof $scope.scans[0][column] === 'object') {
+                    return false;
+                } else {
+                    return true;
+                }
             };
 
+
             $scope.sortCol = column => {
                 if (! $scope.columnIsSortable(column)) {
                     return;
                 }
-                scanArgs.sortColumn = column;
-                scanArgs.sortOrder = ! scanArgs.sortOrder;
+                let inList = false;
+                scanArgs.sortColumns.forEach((sortField, i) => {
+                    if (sortField[0] === column) {
+                        inList = true;
+                        sortField[1] = ! sortField[1];
+                        scanArgs.sortColumns.splice(i, 1);
+                        scanArgs.sortColumns.unshift(sortField);
+                    }
+                });
+                if (! inList) {
+                    if ($scope.analysisStatus !== 'allStatuses') {
+                        scanArgs.sortColumns = [[column, true]];
+                    } else {
+                        scanArgs.sortColumns.unshift([column, true]);
+                        if (scanArgs.sortColumns.length > 3) {
```

Comment:
3 should be captured in a const somewhere globally. It is important.

Reply:
[7322d19](https://github.com/radiasoft/sirepo/pull/7163/commits/7322d198d8db767467b2dfee55a74b6b9b0cba27)